Makefile

```diff
@@ -105,7 +105,7 @@ test-coverage:
 
 .PHONY: test-coverage-with-isb
 test-coverage-with-isb:
-	go test -v -timeout 7m -covermode=atomic -coverprofile=test/profile.cov -tags=isb_redis $(shell go list ./... | grep -v /vendor/ | grep -v /numaflow/test/ | grep -v /pkg/client/ | grep -v /pkg/proto/ | grep -v /hack/)
+	go test -timeout 7m -covermode=atomic -coverprofile=test/profile.cov -tags=isb_redis $(shell go list ./... | grep -v /vendor/ | grep -v /numaflow/test/ | grep -v /pkg/client/ | grep -v /pkg/proto/ | grep -v /hack/)
```

Comment:
What is the reason of disabling `-v`?

Reply:
I found it difficult to identify the failed test in previous commit when Go unit test was failing, since it outputs too many logs: https://github.com/numaproj/numaflow/actions/runs/12943800158/job/36103700507?pr=2357
Looking it again, searching for `--- FAIL` works. Will revert this change.